npc/kafras/functions_kafras.txt

```diff
@@ -181,6 +199,58 @@ function	script	F_Kafra	{
 			callfunc "F_KafInfo",getarg(2);
 			next;
 		}
+		else if (.@K_Menu0$[.@j] == "~ Prontera Market ~") {
+			.@npc_name$ = strnpcinfo(0);
+			mes "[Kafra Employee]";
+			mes "I will guide you to Prontera Market.";
+			close2;
+			if (.@npc_name$ == "Kafra Employee#6")
```

Comment:
How about using the unique name `kaf_prontera`, `kaf_prontera2`.. then revert the `#6`